### PR TITLE
Footer fetch

### PR DESCRIPTION
**Pull request by Emilyempa**
Fetching data from API.JSON for footer

### CODE REVIEW

File: footer.js
Comment (Nishune) [inline footer.js:1]: Mhh, i assume this is where the fetch should happen but the file is empty?

**Review by Nishune — CHANGES_REQUESTED** [review]
Im not sure if this is correct? seems like the fetch is not in your JS file? Also i think you should put the JS file in the index folder under pages

File: footer.js
Comment (Emilyempa) [reply footer.js:1]: fixed now..may have been dedleted when fixing merge conflict or something

File: footer.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+(async function startFooter() {
```
Comment (Nishune) [inline footer.js:1]: This looks good now, good work! =)

**Review by Nishune — APPROVED** [review]

**Pull request merged**